### ISSUE TITLE
error after compile

### ISSUE THREAD

**Issue by danielguerra69** [blacktop/docker-zeek#2]
Hi,

It still compiles but there are some openssl issues.
After running bro I get this error.
Error loading shared library libssl.so.1.0.0: No such file or directory (needed by /usr/local/bro/bin/bro)
Error loading shared library libcrypto.so.1.0.0: No such file or directory (needed by /usr/local/bro/bin/bro)
Error loading shared library libssl.so.1.0.0: No such file or directory (needed by /usr/local/bro/lib/libcaf_openssl.so.0.16.3)
Error loading shared library libcrypto.so.1.0.0: No such file or directory (needed by /usr/local/bro/lib/libcaf_openssl.so.0.16.3)

Regards,
Daniel

**Comment by danielguerra69** [comment]
I checked the alpine packages, result.
https://pkgs.alpinelinux.org/contents?file=libssl.so.1.0.0&path=&name=&branch=v3.8
Alpine says its there...
Inside the container.
find / -name libssl.so.1.*
/lib/libssl.so.1.1
/usr/lib/libssl.so.1.1
I tried to symlink libssl.so.1.0.0 to libssl.so.1.1, but same error.
Its the same for libcrypto

**Comment by danielguerra69** [comment]
Line 58 in the Dockerfile
FROM alpine:latest
to
FROM alpine:3.8
Problem solved !

**Comment by blacktop** [comment]
hmmm it looks like the "new" latest is `3.9` and that breaks ssl? 😱 that's weird?  You might also want to let zeek know, because they support building on alpine

**Comment by blacktop** [comment]
I have fixed it.  Thank you for letting me know! 👍

**Comment by danielguerra69** [comment]
The only way to advance is to stick to version numbers ;)

**Comment by blacktop** [comment]
I am using alpine:3.10 in my 3.0 zeek image.

Can you please test it that it doesn't have the same issues that latest/3.9 alpine had?

**Comment by danielguerra69** [comment]
Hi,
Works perfect 👍
I saw you have the version numbers fixed.
I tested it with a pcap I know very well and
the results are great. Finally ntp working :D
Successfully built 54e9e7ed1d96
Successfully tagged zeek:latest

Image info
zeek                     latest              54e9e7ed1d96        7 minutes ago       39MB

(off topic)
I have lots of problems, with alpine in common, on ubuntu hosts running in vmware. It complains about libc problems. This vmware is configured very strict, still I wonder which libc call causes it.
